libs/pages/application/src/lib/feature/page-settings-resources-feature/page-settings-resources-feature.tsx

```diff
@@ -38,8 +38,7 @@ export function PageSettingsResourcesFeature() {
       a?.memory === b?.memory &&
       a?.cpu === b?.cpu &&
       a?.min_running_instances === b?.min_running_instances &&
-      a?.max_running_instances === b?.max_running_instances &&
-      JSON.stringify(a?.instances) === JSON.stringify(b?.instances)
```

Comment:
I don't think this is necessary to compare instance to retrieve the right app from the state